src/OpenTelemetry.Instrumentation.AWSLambda/OpenTelemetry.Instrumentation.AWSLambda.csproj

```diff
@@ -10,13 +10,15 @@
   <ItemGroup>
     <PackageReference Include="Amazon.Lambda.APIGatewayEvents" Version="2.4.1" />
     <PackageReference Include="Amazon.Lambda.Core" Version="1.2.0" />
+    <PackageReference Include="Amazon.Lambda.SNSEvents" Version="2.0.0" />
+    <PackageReference Include="Amazon.Lambda.SQSEvents" Version="2.1.0" />
```

Comment:
We have all-different versions here. Probably we should stick to the lowest possible for max compatiblity? Or are these independently versioned?

Reply:
It's the latest stable versions available at the moment. There is no 2.0.1 available for the SNSEvents. 
https://www.nuget.org/packages/Amazon.Lambda.SQSEvents#versions-body-tab
https://www.nuget.org/packages/Amazon.Lambda.SNSEvents#versions-body-tab

Do you think we still have to keep versions equal ?

Reply:
No, I agree if they aren't released together, it does not make any sense to keep them in sync.